maps/ministation/uplink_item.dm

```diff
@@ -20,7 +20,7 @@
 	..()
 	item = null // Disabled
 
-/datum/uplink_item/stealthy_tools/syndigolashes/New()
+/datum/uplink_item/stealthy_tools/syndigaloshes/New()
```

Comment:
```  /datum/uplink_item/nukeprice/syndigolashes ```

see above line in same file and edit.  also careful now you're changing paths not desc.